nextcord/guild.py

```diff
@@ -2523,7 +2532,7 @@ async def create_role(
         colour: Union[Colour, int] = ...,
         hoist: bool = ...,
         mentionable: bool = ...,
-        icon: Optional[Union[str, bytes, File]] = ...,
+        icon: Optional[Union[str, bytes, File, Asset, Attachment]] = ...,
```

Comment:
For consistency, maybe `File` could be removed from role icon parameters?

I would be fine with that breaking change as I don't see any reason someone would need to pass a `File`. It's more of just a utility class for sending files in messages.

I'm fine with leaving things the way they are though, and I'm open to hearing other opinions on this.

Reply:
I also thought about it. I was confused why File was there in the first place...
Maybe because Asset and Attachment wasn't there